### ISSUE TITLE
Unable to log into keybase

### ISSUE THREAD

**Issue by Bezoar** [keybase/keybase-issues#2953]
I removed my earlier key pair and reset my password. However, whenever I try to log in from the GUI, I get "There was an error provisioning" with "Unknown error: DB error" underneath.

I then tried to log in with the command line tool, and got:

mw@laptopotamus:~ [543] $ keybase login
Your keybase username or email address: bezoar@gmail.com
Enter a public name for this device: laptop
▶ ERROR DB error (error 501)

I saw from another issue that you had to reset something server-side. Is that the case here? If not, what should I do to further troubleshoot this issue?

Thanks.

**Comment by hashhar** [comment]
I had the same issue today on a new machine.

**Comment by maxtaco** [comment]
This is a new issue as far as I know. Can you send a 'keybase log send'?

I will look into it tonight if I can.

**Comment by proxa** [comment]
Same thing here.  Fresh install.  I just signed up

```
Enter a public name for this device: laptop
▶ ERROR DB error (error 501)
zsh: exit 2     keybase login
```
Log ID: 52a2085bc46d1abca8d98a1c

**Comment by maxtaco** [comment]
On it.

Cc @oconnor663

**Comment by hashhar** [comment]
Here's my log id: 271a323e2503abbb33d4da1c

I intially thought it was because I wasn't using the latest client (on ArchLinux), but the latest client gives the same error.

There's an additional detail that may be helpful: I had reset my account before starting this process.

**Comment by Bezoar** [comment]
My log id is a27027ea17ef531898e1571c - thank you for looking into this.

**Comment by maxtaco** [comment]
Just fixed it. Try again?

**Comment by hashhar** [comment]
Works.

**Comment by proxa** [comment]
Can confirm fixed

**Comment by Bezoar** [comment]
Works for me, thanks.

**Comment by bradwood** [comment]
I'm still having this issue... Log ID: 885bfa3776fab86ec5530e1c

**Comment by maxtaco** [comment]
@bradwood it seems like your passphrase is long and the OSX keychain threw away your saved password hash.  Does it work to log into the web site?
